internal/oidc/provider.go

```diff
@@ -75,7 +76,7 @@ func NewOpenIDConnectProvider(configuration *schema.OpenIDConnectConfiguration)
 		compose.OAuth2AuthorizeImplicitFactory,
 		compose.OAuth2ClientCredentialsGrantFactory,
 		compose.OAuth2RefreshTokenGrantFactory,
-		compose.OAuth2ResourceOwnerPasswordCredentialsFactory,
+		// compose.OAuth2ResourceOwnerPasswordCredentialsFactory,
```

Comment:
I don't actually recall disabling it, so will reenable it and see if it breaks something. I think it was related to us not implementing a required interface.

Reply:
Yeah so the method is meant to allow username and password to obtain an resources, however we never properly implemented this and it would always fail. I think we should remove it since it wasn't working anyway and implement it with policies or at least a toggle to turn it on.